_DeploymentAndDistroPackaging/ansible/roles/ciao-controller/tasks/certificates.yml

```diff
@@ -43,6 +43,8 @@
         creates: cert-CNCIAgent-localhost.pem
       - role: controller
         creates: cert-Controller-localhost.pem
+      - role: agent,netagent
+        creates: cert-CNAgent-NetworkingAgent-localhost.pem
```

Comment:
Yes, the cert is created and stays on the deployment machine. It will only be deployed to the launcher node if that launcher node is present in both [networks] and [computes] sections